src/misc/test/msg.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { msg } from "../msg";
+import { publish } from "../../pubSub/pubSub";
+
+describe("msg test", () => {
```

Comment:
🤔

Reply:
Test cases improved at edf3406, thanks for the feedback :)